guides/examples/basic-web-app.md

```diff
@@ -71,20 +105,27 @@ def hello_world():
 {% endcode %}
 
 Lines 12 and 13 are where we establish out connection to the database, at this point it's
-simply pointing to localhost. If you're running a local postgres instance then starting
+simply pointing to `localhost`. If you're running a local postgres instance then starting
 this Flask app will now show you how many times you've visited it, storing each new visit
 in the database.
 
-Notes to run the example:
-- You also need to add following Python dependencies by running:
-  - Flask: `pip3 install flask`
-  - psycopg2: `pip3 install psycopg2-binary`
-- To run this Flask app (`main.py`) use: `flask --app main run`
+Before doing any Ockam related tasks, feel free to run the Python script now by following
+the instructions below:
+1. You also need to add following Python dependencies by running:
+    - `Flask`: `pip3 install flask`
+    - `psycopg2`: `pip3 install psycopg2-binary`
+2. To run this `Flask` app (`main.py`) type:
+    - `flask --app main run`
+3. To see it running in a web browser open this URL: `http://127.0.0.1:5000/`
 
 ### Moving the database
 
-Now we can add Ockam into the mix 🎉. Lets change the port that the database server is
-listening on to `5433` (_the default port is `5432`_). This will ensure that we're not
+> In our story about our imaginary company and team of three, the following steps would
+> have been performed by Akira, our database admin. You can simply follow these steps on
+> your machine.
+
+Before we can add Ockam into the mix, first lets change the port that the database server
```

Comment:
i agree 100p. ive made the changes so that u no longer have to move the db :). 1 line in
the main.py needs to be changed. lots of explanations are provided that this port number
shud be loaded from machine configuration rather than hardcoded anyway; it is hardcoded
just for the purposes of making this exercise simpler